src/AppBundle/DataFixtures/ORM/LoadFixtures.php

```diff
@@ -0,0 +1,194 @@
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Defines the sample data to load in the database when running the unit and
+ * functional tests. Execute this command to load the data:
+ *
+ *   $ php app/console doctrine:fixtures:load
+ *
+ * See http://symfony.com/doc/current/bundles/DoctrineFixturesBundle/index.html
+ *
+ * @author Ryan Weaver <weaverryan@gmail.com>
+ * @author Javier Eguiluz <javier.eguiluz@gmail.com>
+ */
+class LoadFixtures implements FixtureInterface, ContainerAwareInterface
+{
+    /** @var ContainerInterface */
+    private $container;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function load(ObjectManager $manager)
+    {
+        $this->loadUsers($manager);
+        $this->loadPosts($manager);
+    }
+
+    private function loadUsers(ObjectManager $manager)
+    {
+        $passwordEncoder = $this->container->get('security.password_encoder');
+
+        $johnUser = new User();
+        $johnUser->setUsername('john_user');
+        $johnUser->setEmail('john_user@symfony.com');
+        $encodedPassword = $passwordEncoder->encodePassword($johnUser, 'kitten');
+        $johnUser->setPassword($encodedPassword);
+        $manager->persist($johnUser);
+
+        $annaAdmin = new User();
+        $annaAdmin->setUsername('anna_admin');
+        $annaAdmin->setEmail('anna_admin@symfony.com');
+        $annaAdmin->setRoles(array('ROLE_ADMIN'));
+        $encodedPassword = $passwordEncoder->encodePassword($annaAdmin, 'kitten');
+        $annaAdmin->setPassword($encodedPassword);
+        $manager->persist($annaAdmin);
+
+        $manager->flush();
+    }
+
+    private function loadPosts(ObjectManager $manager)
+    {
+        foreach (range(1, 30) as $i) {
+            $post = new Post();
+
+            $post->setTitle($this->getRandomPostTitle());
+            $post->setSummary($this->getRandomPostSummary());
+            $post->setSlug($this->container->get('slugger')->slugify($post->getTitle()));
+            $post->setContent($this->getPostContent());
+            $post->setAuthorEmail('anna_admin@symfony.com');
+            $post->setPublishedAt(new \DateTime('now - '.$i.'days'));
+
+            foreach (range(1, 5) as $j) {
+                $comment = new Comment();
+
+                $comment->setAuthorEmail('john_user@symfony.com');
+                $comment->setPublishedAt(new \DateTime('now + '.($i + $j).'seconds'));
+                $comment->setContent($this->getRandomCommentContent());
+                $comment->setPost($post);
+
+                $manager->persist($comment);
+                $post->addComment($comment);
+            }
+
+            $manager->persist($post);
+        }
+
+        $manager->flush();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setContainer(ContainerInterface $container = null)
```

Comment:
__[development][code_sniffer]__
* Declare public methods first,then protected ones and finally private ones

<!-- Comment by [SideCI](https://sideci.com) -->